pkg/controller/nodes/task/sleeper_plugin.go

```diff
@@ -0,0 +1,61 @@
+package task
+
+import (
+	"context"
+	pluginMachinery "github.com/flyteorg/flyteplugins/go/tasks/pluginmachinery"
+	pluginCore "github.com/flyteorg/flyteplugins/go/tasks/pluginmachinery/core"
+	"github.com/flyteorg/flytestdlib/logger"
+	"time"
+)
+
+type SleeperPlugin struct {
```

Comment:
Can we add one that simulates K8s APIs? I think that might be more useful to start with...